solutions/standard/stack_definition.json

```diff
@@ -406,6 +406,32 @@
         }
       ]
     },
+    {
+      "name": "Essential Security - Security Compliance Center Agents",
+      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.b1d07357-ed1f-4e64-a6e4-a500844f1941-global",
+      "inputs": [
+        {
+          "name": "ibmcloud_api_key",
+          "value": ""
+        },
+        {
+          "name": "cluster_id",
+          "value": "ref:./members/Workload - Compute Red Hat OpenShift Container Platform on VPC/outputs/workload_cluster_id"
```

Comment:
is that syntax correct? Looking at other refs in the code, it uses syntax like this:

`ref:../Workload - Compute Red Hat OpenShift Container Platform on VPC/outputs/workload_cluster_id`